app/view_models/changelog_admin/changelog_entry_action.rb

```diff
@@ -0,0 +1,44 @@
+module ChangelogAdmin
+  class ChangelogEntryAction
+    def self.all
+      [
+        ChangelogEntryAction.new(name: :publish, confirm: true, method: :post),
+        ChangelogEntryAction.new(name: :unpublish, confirm: true, method: :post),
```

Comment:
Can we comment this out for now pls?

Reply:
Yeah. Makes sense. I'll comment out and merge.